mod-audit-server/src/main/java/org/folio/services/acquisition/OrderAuditEventsService.java

```diff
@@ -15,4 +19,12 @@ public interface OrderAuditEventsService {
    * @return successful future if event has not been processed, or failed future otherwise
    */
   Future<RowSet<Row>> saveOrderAuditEvent(OrderAuditEvent orderAuditEvent, String tenantId);
+
+  /**
+   * Searches for OrderAuditEvent by id
+   *
+   * @param id OrderAuditEvent id
+   * @return future with optional OrderAuditEvent
+   */
+  Future<OrderAuditEventCollection> getAcquisitionOrderEventById(String id, int limit, int offset, String tenantId);
```

Comment:
Need to apply the same method convention as for dao

We need to rename this method like this:
getAuditEventsByOrderId(String orderId, int limit, int offset, String tenantId);

Also need to rename java doc because we can return multiple events:
Searches for order audit events by order id

Reply:
fixed